src/AstSemanticChecker.cpp

```diff
@@ -699,28 +710,23 @@ void AstSemanticCheckerImpl::checkClause(const AstClause& clause) {
         checkFact(clause);
     }
 
-    // check for use-once variables
+    // check _<var> occurrences
```

Comment:
The explanatory comment could be clearer, e.g.,
```suggestion
    // check _<var> variables are unique and uniquely named
```